.changeset/dry-eyes-scream.md

```
@@ -0,0 +1,12 @@
+---
+'@mastra/core': minor
+---
+
+Added a strict advanced trace-query contract, trusted planner, cursor binding, and storage capability.
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)